app/assets/stylesheets/components/_navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--yellow);
+  position: fixed;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 100vw;
+  height: 100px;
+}
+
+.logo {
+  font-family: eds-market-bold-slant, sans-serif;
+  font-size: var(--font-size-s);
+  line-height: 1.25;
+  color: var(--blue);
+  text-align: center;
+}
+
+.design-elements {
+  font-family: Eds Market Design Elements;
+}
```

Comment:
**question:** Are we using this class somewhere?

Reply:
no, I was going to, but didn't. I'm going to delete it.